src/templates/smfcfg.yaml.j2

```diff
@@ -34,29 +34,6 @@ configuration:
     addr: {{ pod_ip }}
   nrfUri: {{ nrf_url }}
 
-# the kind of log output
-# debugLevel: how detailed to output, value: trace, debug, info, warn, error, fatal, panic
-# ReportCaller: enable the caller report or not, value: true or false
 logger:
   SMF:
-    debugLevel: info
-    ReportCaller: false
-  NAS:
-    debugLevel: info
-    ReportCaller: false
-  NGAP:
-    debugLevel: info
-    ReportCaller: false
-  Aper:
-    debugLevel: info
-    ReportCaller: false
-  PathUtil:
-    debugLevel: info
-    ReportCaller: false
-  OpenApi:
-    debugLevel: info
-    ReportCaller: false
-  PFCP:
-    debugLevel: info
-    ReportCaller: false
-
+    debugLevel: {{ log_level }}
```

Comment:
I see you remove the PFCP section, wouldn't it be useful in the SMF?

Reply:
As far as I can see re-adding this section won't change anything. I don't see ant reference in the upstream code that will set the log level for PFCP.

https://github.com/omec-project/smf/blob/main/service/init.go

Reply:
Alright, just wanted to make sure we would not miss logs.